ci/environment-dev-3.6.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
   - numpy
   - pandas
   - scipy
-  - xarray>=0.16.0
+  - xarray=0.16.0
```

Comment:
I think you need `xarray==0.16.0` rather than one `=` sign. Did we figure out what is causing the break from `0.16.1`? I'm concerned about this being a big incompatibility with some user environments. Looks like it was an `apply_ufunc` thing in `xskillscore` for instance..

Reply:
Yes so for now we should use xs master